### ISSUE TITLE
Slow to expand on FF when on a slow network

### ISSUE THREAD

**Issue by dylans** [wkeese/old-api-viewer#55]
When on a hotel network or 4G network, clicking on Dojo or DojoX is very slow to open (10-20 seconds), and provides no visual feedback that it is loading. Chrome by comparison opens right away.

**Comment by wkeese** [comment]
You're comparing "DojoX" to "Chrome"?   It doesn't make sense.

**Comment by dylans** [comment]
No, I'm saying it's slow to expand/collapse on FF (see the ticket title), but fast on Chrome. I notice this when I'm on a slow network connection only.

**Comment by wkeese** [comment]
Hmm, that's odd.   It fetches all the data when the tree is initially displayed.  Perhaps it's an issue downloading the icons.  What does firebug's NET tab show you?

Since it's limited to slow network connections, I don't think it's an issue with the animation.

**Comment by kitsonk** [comment]
Also, could it have been that the tree data was cached in Chrome, but not in FF, requiring a load?

**Comment by wkeese** [comment]
Maybe... except then, it should be slow to display the initial tree, not to open the dojox branch.   Because it's downloading the data all at once.  (Even though it shouldn't be.)
